projects/plugins/jetpack/_inc/client/components/jetpack-product-card/index.jsx

```diff
@@ -0,0 +1,159 @@
+/**
+ * External dependencies
+ */
+import React, { useCallback, useEffect } from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import { getCurrencyObject } from '@automattic/format-currency';
+import { __, sprintf } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import analytics from 'lib/analytics';
+import Button from 'components/button';
+import Gridicon from 'components/gridicon';
+
+/**
+ * Style dependencies
+ */
+import './style.scss';
+
+const formatPrice = ( price, currencyCode, isOldPrice = false ) => {
+	const priceObject = getCurrencyObject( price, currencyCode );
+	const classes = classNames( {
+		'jp-product-card__raw-price': true,
+		'jp-product-card__raw-price--is-old-price': isOldPrice,
+	} );
+
+	return (
+		<div className={ classes }>
+			<sup className="jp-product-card__currency-symbol">{ priceObject.symbol }</sup>
+			<span className="jp-product-card__price-integer">{ priceObject.integer }</span>
+			<sup className="jp-product-card__price-fraction">{ priceObject.fraction }</sup>
+		</div>
```

Comment:
I don't think it does because even though the passed value is `countryCode` (e.g.: `USD`), we actually display the symbol instead (e.g.: `$`).
So for danish currency it would say `kr` which is the same no matter the language (even though I personally prefer DKR to differentiate between e.g. Swedish Kr and Danish Kr 😛 ).

I've copied the logic from cloud.jetpack.com where they do it the same way without translating it (not saying this is a stamp of approval, but I don't know of any currency symbol issues over there).